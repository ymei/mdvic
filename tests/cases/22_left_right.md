Inline: $\left( \frac{a}{b} \right)$ and $\left\langle x \right\rangle$.

Display:
$$\left( \frac{a}{b} \right)$$

