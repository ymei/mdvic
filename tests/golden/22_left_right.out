Inline: ( a⁄b ) and ⟨ x ⟩.

Display: 
( 
a
-
b )
